### ISSUE TITLE
Use iTwinUI keyboard components

### ISSUE THREAD

**Issue by GerardasB** [iTwin/appui#126]
Investigate using iTwinUI keyboard components in favor of custom implementation in `ToolAssistanceField`.

![Screenshot 2024-03-29 at 12 09 34](https://github.com/iTwin/appui/assets/10091419/d3c52fe7-3861-4d1a-9912-a89ce759d17d)

Update: current `KeyboardShortcutMenu` implementation displays shortcuts as underlined symbols (example below). It can be changed to iTwinUI `Kbd` component.

![keyboard-shortcut-menu-test-1-chromium-win32](https://github.com/iTwin/appui/assets/138560995/c147e4ab-9b49-4362-ae45-32ef099c66f0)

**Comment by mayank99** [comment]
I don't think this should use `<Kbd>`. That component is intended to be used alongside other static text, e.g. for documentation purposes or to indicate keyboard shortcuts in a **Help** page.

A better approach would be to show these as static, muted, monospace text. Ideally right aligned like in macOS menubar:

![image](https://github.com/iTwin/appui/assets/9084735/e08c4463-51fc-4ec2-b83d-71563849c99a)

I guess right aligned becomes difficult if you also have to show an arrow/caret, so you could show it as sublabel underneath the main label, though it wouldn't look as clean.

**Comment by GerardasB** [comment]
> I don't think this should use `<Kbd>`. That component is intended to be used alongside other static text, e.g. for documentation purposes or to indicate keyboard shortcuts in a **Help** page.

Good point, I agree about the menu. I've updated the issue description w/ image of `ToolAssistanceField`, which I think is a good candidate for  `<Kbd />`.
